### PR TITLE
set up TOC for tutorials and guide. Create sample doc

### PR DESCRIPTION
**Pull request by tafsiri**
Adds a sample markdown file and initial organization and TOC for tutorials and guides. Most links are placeholders and link to the same file.

Tested with local devsite build.

cc @lamberta in case you see something obviously wrong with _book.yaml.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-website/233)

### CODE REVIEW

File: docs/_book.yaml [hunk 1]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
     other:
     - name: Tutorials
       contents:
-      - title: Tutorials overview
-        path: /js/tutorials/
-      - title: Baz tutorial
-        path: /js/tutorials/baz_tutorial
+      - title: Getting Started
+        path: /js/tutorials/index
+      - title: Getting Setup
```
Comment (lamberta) [inline docs/_book.yaml]: Either "Setup" or "Install"

File: docs/_book.yaml [hunk 2]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/tutorials/setup
+
+      - heading: Training Models
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - heading: Train models
```

File: docs/_book.yaml [hunk 3]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+      - title: Fitting a Curve to Two-Dimensional Data
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Fit a curve to two-dimensional data
```

If that doesn't fit, try: Fit a curve to 2D data

File: docs/_book.yaml [hunk 4]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/tutorials/training/curve
+      - title: Recognizing Handwritten Digits with CNNs
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Recognize handwritten digits with CNNs
```

File: docs/_book.yaml [hunk 5]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/tutorials/training/mnist
+      - title: Saving and Loading Models
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Save and load models
```

File: docs/_book.yaml [hunk 6]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/tutorials/training/mnist
+
+      - heading: Transfer Learning
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - heading: Transfer learning
```

File: docs/_book.yaml [hunk 7]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+      - title: Building a webcam based game controller
+        path: /js/sample
+      - title: Building an Audio Recognizer
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Build an audio recognizer
```

File: docs/_book.yaml [hunk 8]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/sample
+
+      - heading: Importing Python Models
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - heading: Import Python models
```

File: docs/_book.yaml [hunk 9]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+      - title: Importing a Keras Model
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Import a Keras model
```

File: docs/_book.yaml [hunk 10]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/sample
+      - title: Importing a TensorFlow SavedModel
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Import a TensorFlow SavedModel
```

File: docs/_book.yaml [hunk 11]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/sample
+
+      - heading: Learning Resources
+      - title: Machine Learning
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Machine learning
```

File: docs/_book.yaml [hunk 12]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
-      - title: Foo guide
-        path: /js/foo_guide
+      - heading: Introduction
+      - title: Overview
+        path: /js/guides/index
+      - title: FAQ
+        path: /js/sample
+
+      - heading: Fundamentals
+      - title: Tensors & Operations
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Tensors and operations
```

File: docs/_book.yaml [hunk 13]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/guides/index
+      - title: Backends
+        path: /js/sample
+
+      - heading: Models
+      - title: Models & Layers
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Models and layers
```

File: docs/_book.yaml [hunk 14]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/sample
+      - title: Training Models
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Train models
```

Training is also ok here

File: docs/_book.yaml [hunk 15]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/sample
+      - title: Model Conversion
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Model conversion
```

File: docs/_book.yaml [hunk 16]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
...
+        path: /js/sample
+
+      - heading: Node.js
+      - title: Choosing between Node.js and the browser
+        path: /js/sample
+      - title: Choosing the right Node.js binding
+        path: /js/sample
+
+      - heading: Visualization
+      - title: Visualizing Models in the Browser
```
Comment (lamberta) [inline docs/_book.yaml]:
```suggestion
      - title: Visualizing models in the browser
```

File: docs/_book.yaml [hunk 17]
```diff
@@ -13,22 +13,74 @@ upper_tabs:
     other:
     - name: Tutorials
       contents:
-      - title: Tutorials overview
-        path: /js/tutorials/
-      - title: Baz tutorial
-        path: /js/tutorials/baz_tutorial
+      - title: Getting Started
```
[the remark this answers is not in the document]
Comment (tafsiri) [reply docs/_book.yaml]: Done

File: docs/_book.yaml (hunk 1 above)
Comment (tafsiri) [reply docs/_book.yaml]: Going with setup

File: docs/_book.yaml (hunk 3 above)
Comment (tafsiri) [reply docs/_book.yaml]: Done

File: docs/_book.yaml (hunk 5 above)
Comment (tafsiri) [reply docs/_book.yaml]: done

File: docs/_book.yaml (hunk 7 above)
Comment (tafsiri) [reply docs/_book.yaml]: done

File: docs/_book.yaml (hunk 6 above)
Comment (tafsiri) [reply docs/_book.yaml]: done

File: docs/_book.yaml (hunk 8 above)
Comment (tafsiri) [reply docs/_book.yaml]: Done

File: docs/_book.yaml (hunk 10 above)
Comment (tafsiri) [reply docs/_book.yaml]: Done

File: docs/_book.yaml (hunk 11 above)
Comment (tafsiri) [reply docs/_book.yaml]: Done

File: docs/_book.yaml (hunk 12 above)
Comment (tafsiri) [reply docs/_book.yaml]: Done

File: docs/_book.yaml (hunk 14 above)
Comment (tafsiri) [reply docs/_book.yaml]: Dine

File: docs/_book.yaml (hunk 13 above)
Comment (tafsiri) [reply docs/_book.yaml]: done

File: docs/_book.yaml (hunk 16 above)
Comment (tafsiri) [reply docs/_book.yaml]: Done

File: docs/_book.yaml (hunk 15 above)
Comment (tafsiri) [reply docs/_book.yaml]: Done

**Pull request merged**